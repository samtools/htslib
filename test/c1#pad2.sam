@SQ	SN:c1	LN:50
s0a	0	c1	1	0	10M	*	0	0	AACCGCGGTT	*
s0b	0	c1	1	0	10M	*	0	0	AACCGCGGTT	*
s0c	0	c1	1	0	10M	*	0	0	AACCGCGGTT	*
s0d	0	c1	1	0	10M	*	0	0	AACCGCGGTT	*
s1	0	c1	1	0	5M6I5M	*	0	0	AACCGGTTAACCGGTT	*
s2	0	c1	1	0	5M1P4I1P5M	*	0	0	AACCGTTAACGGTT	*
s3	0	c1	1	0	5M3I3P5M	*	0	0	AACCGGTTCGGTT	*
s4	0	c1	1	0	5M3P3I5M	*	0	0	AACCGAACCGGTT	*
s5	0	c1	1	0	4M1D2P2I2P1D4M	*	0	0	AACCTAGGTT	*
s6	0	c1	1	0	2M3D6I3D2M	*	0	0	AAGTTAACTT	*
s7	0	c1	1	0	4M2D4M	*	0	0	AACCGGTT	*
s8	0	c1	1	0	5D2P2I2P5D	*	0	0	TA	*
s9	0	c1	5	0	1M2P2I2P	*	0	0	GTA	*
