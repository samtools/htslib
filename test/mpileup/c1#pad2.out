c1	1	12	^!A^!A^!A^!A^!A^!A^!A^!A^!A^!A^!A^!*	~~~~~~~~~~~~
c1	2	12	AAAAAAAAAA-3()A*	~~~~~~~~~~~~
c1	3	12	CCCCCCCCC*C*	~~~~~~~~~~~~
c1	4	12	CCCCCCCCC-1()*C-2()*	~~~~~~~~~~~~
c1	5	13	GGGGG+6(GTTAAC)G+6(*TTAA*)G+6(GTT***)G+6(***AAC)*+6(**TA**)-1()*+6(GTTAAC)-3()**+6(**TA**)-5()^!G+6(**TA**)$	~~~~~~~~~~~~~
c1	6	12	CCCCCCCC****	~~~~~~~~~~~~
c1	7	12	GGGGGGGGG*G*	~~~~~~~~~~~~
c1	8	12	GGGGGGGGG*G*	~~~~~~~~~~~~
c1	9	12	TTTTTTTTTTT*	~~~~~~~~~~~~
c1	10	12	T$T$T$T$T$T$T$T$T$T$T$*$	~~~~~~~~~~~~
